docs/en_US/install/3ds.md

```diff
@@ -5,6 +5,13 @@
 	SYSTEM TRANSFERS ARE NOT CURRENTLY SUPPORTED BY OUR SERVERS. ATTEMPTING TO PERFORM A SYSTEM TRANSFER MAY PREVENT YOU FROM BEING ABLE TO GO ONLINE IN THE FUTURE. SUPPORT FOR SYSTEM TRANSFERS IS IN DEVELOPMENT.
 </div>
 
+<div class="tip red">
+	<strong>CAUTION:</strong>
+	If you try and boot up the 3DS Badge Arcade while on Pretendo server, your badges will be wiped! If you have badges you do not want to be wiped, please do not load Badge Arcade until further notice or make a backup of your badges by going to the folder "sd:Nintendo 3DS\[ID0][ID1]\extdata\00000000\000014d1" and copying them to a safe place.
```

Comment:
The grammar here is a little rough, and the path is incorrect. We also prefer to mark paths, labels on the console, etc, in codeblocks, and the path formatting is not consistent with the rest of the page. And finally, these callouts should be as simple to digest as possible. Some of this information does not need to be here, is it's duplicated later in the page. Something like:

```suggestion
	Launching Nintendo Badge Arcade while connected to one network with badges obtained while connected to the other will cause your local save data to not match your server save data. This will cause your badges to disappear.
```

To condense the information down. Also it applies both ways (badges obtained from our servers will be removed when launching NN), so that should be made clear. The example I proposed isn't perfect but it gets the point across. @hauntii @gitlimes thoughts?

Reply:
Thanks for the suggestions! I'll start adding them in right now, thanks a ton Jon.

Reply:
I'll keep this open so the other can reply here.

Reply:
Also, reason why the codeblocks were not there is due to GitHub not letting me put them in correctly? I guess you doing it did the trick.

Reply:
> GitHub not letting me put them in correctly?

Are you making these changes directly in the GitHub UI? Please test your changes before making any PRs